cmake/kconfig.cmake

```diff
@@ -228,6 +228,14 @@ if(EXISTS ${BOARD_DEFCONFIG})
 
         # platform-specific include path
         include_directories(${PX4_SOURCE_DIR}/platforms/${PX4_PLATFORM}/src/px4/common/include)
+
+        if("${PX4_BOARD}" MATCHES "modalai_voxl2")
```

Comment:
To be honest this inclusion is pretty bad.

At a minimum could it be moved to `boards/modalai/voxl2/CMakeLists.txt` which is already included automatically?

I would also think most (but probably not all) of what's in voxl2_posix.cmake and voxl2_qurt.cmake really belongs under platforms/ with appropriate kconfig options for things like ORB_COMMUNICATOR, CONFIG_PARAM_SERVER, etc. We can figure this out incrementally though.

Reply:
The voxl2_posix.cmake definitely wasn't needed there so I removed it. Still looking at best options for qurt though.